### PR TITLE
Add sync modes for Smart Proxy Servers

### PR DESCRIPTION
**Pull request by maximiliankolb**
Tested on Foreman 3.9:

image

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.9/Katello 4.11 (planned Satellite 6.15)
* [x] Foreman 3.8/Katello 4.10
* [x] Foreman 3.7/Katello 4.9 (Satellite 6.14)

### CODE REVIEW

**Comment by maximiliankolb** [comment]
cc @m-bucher

File: guides/common/modules/proc_synchronizing-an-external-smart-proxy.adoc [hunk 1]
```diff
@@ -1,9 +1,14 @@
-[id="synchronizing-an-external-{smart-proxy-context}_{context}"]
-= Synchronizing an External {SmartProxy}
+[id="Synchronizing_a_{smart-proxy-context-titlecase}_Server_{context}"]
+= Synchronizing a {SmartProxyServer}
```
Comment (maximiliankolb) [inline guides/common/modules/proc_synchronizing-an-external-smart-proxy.adoc:2]:
```suggestion
= Synchronizing content from {ProjectServer} to {SmartProxyServers}
```

Should I change the title/anchor/file name?

Comment (adamlazik1) [reply guides/common/modules/proc_synchronizing-an-external-smart-proxy.adoc:2]: Sure.

File: guides/common/modules/proc_synchronizing-an-external-smart-proxy.adoc [hunk 2]
```diff
@@ -1,9 +1,14 @@
...
 
-Synchronize an external {SmartProxy} with {Project}.
+Use this procedure to synchronize content from your {ProjectServer} to your {SmartProxyServer}.
+
+.Prerequisite
+. Ensure that you either select the relevant organization and location context of your {SmartProxyServer} or choose *Any Organization* and *Any Location*.
```
Comment (adamlazik1) [inline guides/common/modules/proc_synchronizing-an-external-smart-proxy.adoc:7]:
```suggestion
* Ensure that you either select the relevant organization and location context of your {SmartProxyServer} or choose *Any Organization* and *Any Location*.
```

File: guides/common/modules/proc_synchronizing-an-external-smart-proxy.adoc [hunk 3]
```diff
@@ -1,9 +1,14 @@
...
 
 .Procedure
-. To synchronize an external {SmartProxy}, select the relevant organization and location in the {ProjectWebUI}, or choose *Any Organization* and *Any Location*.
-. In the {ProjectWebUI}, navigate to *Infrastructure* > *{SmartProxies}* and click the name of the {SmartProxy} to synchronize.
-. On the *Overview* tab, select *Synchronize*.
+. In the {ProjectWebUI}, navigate to *Infrastructure* > *{SmartProxies}*.
+. Select your {SmartProxyServer}.
+. On the *Overview* tab, click *Synchronize*.
+** Select *Optimized Sync* to synchronize content from your {ProjectServer} to your {SmartProxyServer} that bypasses unnecesary steps to speed up performance.
+** Select *Complete Sync* to perform a complete sync from your {ProjectServer} to your {SmartProxyServer} that synchronizes content even if the metadata has not changed.
```
Comment (adamlazik1) [inline guides/common/modules/proc_synchronizing-an-external-smart-proxy.adoc:14]:
```suggestion
* Select *Optimized Sync* to synchronize content from your {ProjectServer} to your {SmartProxyServer} that bypasses unnecesary steps to speed up performance.
* Select *Complete Sync* to perform a complete sync from your {ProjectServer} to your {SmartProxyServer} that synchronizes content even if the metadata has not changed.
```

**Review by adamlazik1 — COMMENTED** [review]
Looks good stylewise, I only have some minor nitpicks.

File: guides/common/modules/proc_synchronizing-an-external-smart-proxy.adoc (hunk 2 above)
Comment (maximiliankolb) [reply guides/common/modules/proc_synchronizing-an-external-smart-proxy.adoc:7]: Applied.

File: guides/common/modules/proc_synchronizing-an-external-smart-proxy.adoc (hunk 3 above)
Comment (maximiliankolb) [reply guides/common/modules/proc_synchronizing-an-external-smart-proxy.adoc:14]: Applied.

File: guides/common/modules/proc_synchronizing-an-external-smart-proxy.adoc (hunk 1 above)
Comment (maximiliankolb) [reply guides/common/modules/proc_synchronizing-an-external-smart-proxy.adoc:2]: Done!

**Comment by maximiliankolb** [comment]
Merged to master and cherry-picked: :heavy_check_mark: 
   02abf8c8a..b40bb2757  3.9 -> 3.9
   4cfb096fa..9017869e2  3.8 -> 3.8
   61d6fcbbd..5f0dbed6a  3.7 -> 3.7

**Pull request merged**